api/Navigator.json

```diff
@@ -2816,6 +2817,54 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "fixed_return_value": {
```

Comment:
@ddbeck As part of this spec change the deprecated `Navigator.plugins` and `Navigator.mimeTypes` properties were changed from returning "real" plugins (including possibly Adobe flash) to returning hard coded values.

Essentially these are both  long deprecated but some code is still using it to determine if inline PDF viewing is supported, by checking for particular mime types being supported and particular mime type names. This is unreliable and exposes fingerprinting information so the spec made two changes:
- add `pdfViewerEnabled` to indicate reliably inline viewing is supported
- make these old properties return a standard set of plugins if PDF viewing is enabled and an empty object if not -  mirroring `pdfViewerEnabled`. The idea being that old sites would continue to work, but actually be more reliable since the set of strings they use are accurately set. Eventually though we hope all will migrate to  `pdfViewerEnabled

Anyway, I have indicated this using a subfeature, currently just for `Navigator.plugins`. Wanted to check if this is the best way. We might alternatively have a subfeature that is expiring instead - ie one that indicates "plugins actually exist" rather than "plugin values are hard coded". Or I could just add a note for all cases indicating the version.

Would appreciate your guidance.

Reply:
> We might alternatively have a subfeature that is expiring instead - ie one that indicates "plugins actually exist" rather than "plugin values are hard coded". Or I could just add a note for all cases indicating the version.

Yes, I think inverting things is the way to go. Ideally, in a few years, we'll end up in a situation where there hard-coded behavior is the only one that matters, at which point we can get rid of any mention of the historic behavior.